posts/2022-07-11-Rustup-1.25.0.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: "Announcing Rustup 1.25.0"
+author: The Rustup Working Group
+---
+
+The rustup working group is happy to announce the release of rustup version 1.25.0. [Rustup][install] is the recommended tool to install [Rust][rust], a programming language that is empowering everyone to build reliable and efficient software.
+
+If you have a previous version of rustup installed, getting rustup 1.25.0 is as easy as closing your IDE and running:
+
+```
+rustup self update
+```
+
+Rustup will also automatically update itself at the end of a normal toolchain update:
+
+```
+rustup update
+```
+
+If you don't have it already, you can [get rustup][install] from the appropriate page on our website.
+
+[rust]: https://www.rust-lang.org
+[install]: https://rustup.rs
+
+## What's new in rustup 1.25.0
+
+This version of Rustup involves a significant number of internal cleanups, both in terms
+of the Rutup code and its documentation. In addition to a lot of work on the codebase
```

Comment:
```suggestion
of the Rustup code and its documentation. In addition to a lot of work on the codebase
```

Reply:
I can't believe I didn't see this myself, thanks Bjorn